deploy/ansible/inventory/hosts_docker_cluster.list

```diff
@@ -0,0 +1,28 @@
+# This is sample inventory file for deployment using docker-compose (multiple nodes of iroha on target hosts)
+# To start with:
+# 1) set group name (not mandatory, you can use already predefined): e.g. [iroha-east].
+# 2) change inventory hostname (not mandatory, you can use already predefined): e.g. iroha-bench1.
+# 3) set IP address to the address of your server ("ansible_host" variable)
+# 4) set "ansible_user", or leave "root"
+# 5) set "key" variable in the following way:
+# key=0 for the first host of the first node in the inventory
+# in playbook/<playbook name>/group_vars/all.yml or playbook/<playbook name>/group_vars/<group name>.yml
+# you can find parameter "nodes_in_group"
```

Comment:
What if `nodes_in_group` doesn't match the difference of [iroha-west][key] - [iroha-east][key]?

Reply:
@bakhtin unfortunately, everything will fail, or, to be honest, some nodes will be ignored. better not to violate simple math.